src/modules/drawer.js

```diff
@@ -310,9 +318,12 @@ class Drawer extends React.Component {
 
                         break;
                 }
-
                 BODY.style.top = `-${scrollPosition}px`;
+
                 DOMUtils.addClassName(BODY, 'drawer-open');
+                if (positionY && maxHeight) {
+                    BODY.style.position = 'inherit';
+                }
```

Comment:
Nitpick; `if(...) { ... }` block should have an empty line before and after.

Reply:
@groberts314 thanks, will correct it on my next PR on react-cm-ui

Reply:
Sounds good.  If it's not going to distract _too_ much or take _too_ long, maybe it's worth fixing all the ESLINT issues in that file at some point.

@morethanfire can weigh in, but in my own personal opinion, I'd _love_ to get the React CM UI codebase to a point where it is certified LINT free and we can run the linters (ES and Sass) as a part of builds, and fail builds if there are linter issues.  We probably will have an easier time of achieving that first that in React CM UI, _long_ before we would get there in HC (`church-management` repo `/client`), where our ESLINT related pile of tech debt is _ginormous!_

Side note: I had a thought about HC client application and ESLINT.  Maybe we can do something like the following:
* Run ESLINT in a mode where it will not fail the build, but will produce an artifact giving us statistics and/or a comprehensive list of the issues (i.e. redirect the output to a file which gets published as an artifact, and ensure the return code of the command is 0).
* "Baseline" it.  Let's say that, as of some point, we have ~ 10,000 errors and ~ 12,000 warnings (I'm making up numbers as an example).  What we can do as try to get it to fail the build only if the number of errors or warnings in the present run exceeds the baseline threshold values (these are build parameters/variables); i.e. it's a failure only if the changeset included in the new build has introduced new problems.  Then we periodically reduce the threshold values as we reduce the volume of legacy lint, slowly improving overall code quality.

Thoughts?